### PR TITLE
Mergeup master -> 1.0.x

### PR DESCRIPTION
**Pull request by underscorgan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,15 @@ flux, and it should not be considered to have a formal API. However, every
 effort is being made to ensure existing tasks/behavior are not broken as we
 continue to iterate and improve upon it.
 
+## Getting Started
```
Comment (e-gris) [inline README.md]: I think this is a fantastic change. It's the best change. Believe me. :)

File: lib/packaging/osx.rb [hunk 2]
```diff
@@ -3,8 +3,13 @@ class << self
     def sign(target_dir = 'pkg')
       use_identity = "-i #{Pkg::Config.osx_signing_ssh_key}" unless Pkg::Config.osx_signing_ssh_key.nil?
 
-      ssh_host_string = "#{use_identity} #{ENV['USER']}@#{Pkg::Config.osx_signing_server}"
-      rsync_host_string = "-e 'ssh #{use_identity}' #{ENV['USER']}@#{Pkg::Config.osx_signing_server}"
+      if Pkg::Config.osx_signing_server =~ /@/
```
Comment (e-gris) [inline lib/packaging/osx.rb]: <insert my normal complaint about "else" statements>

Comment (underscorgan) [reply lib/packaging/osx.rb]: fair, but should probably be fixed in master then merged up since this is just a mergeup

Comment (e-gris) [reply lib/packaging/osx.rb]: Yup, just wanted to comment, not request any actual action.

**Pull request merged**